_ramblings/2018-07-11-building-a-shell-part-1.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Building a Shell - Part 1
+layout: rambling
+draft: true
+---
+
+A shell is one of the best computer interfaces I've ever used.
+It's *declarative*, before being declarative was cool. It's
+*keyboard centric*, making it fast, and reliable to use. Sure, a
+shell isn't the right tool for everything. I'm not going to try
```

Comment:
You can keep as is, with the negative and but I find this sentence a bit too... negative.